### PR TITLE
Added next step links to the folder for ease of accessing

### PR DESCRIPTION
**Pull request by AbhiSaphire**

### CODE REVIEW

File: step1-02/demo/README.md [hunk 1]
```diff
@@ -84,3 +84,5 @@ Open up the browser inspector and target our "all" button. You'll notice that th
 > **Specificity** states that regardless of cascade, the selector with the highest specificity wins
 
 To fix this problem we need to either reduce the specificity of our button styles, or increase the specificity of the selected style. In this situation we will add `.filter` in front of the `.selected` selector, because the selected style only applies to the filter anyway.
+## Next Step
```
Comment (ecraig12345) [inline step1-02/demo/README.md]: Can you add a blank line before the heading in each file to ensure it gets formatted properly?

File: step1-02/demo/README.md [hunk 2]
```diff
@@ -84,3 +84,5 @@ Open up the browser inspector and target our "all" button. You'll notice that th
...
+[EXERCISE](../exercise)
```
Comment (ecraig12345) [inline step1-02/demo/README.md]: Would prefer to have this in title case (Exercise) for consistency

**Comment by ecraig12345** [comment]
I'm going to close this one due to the merge conflicts and the new PR which does something similar.

File: step1-02/demo/README.md (hunk 1 above)
Comment (AbhiSaphire) [reply step1-02/demo/README.md]: Can you explain me exactly what should i do?

**Pull request closed without merging**